Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload/OldVimeoUploader.swift

```diff
@@ -25,17 +25,18 @@
 //
 
 import Foundation
+import VimeoUpload
 
-class NewVimeoUpload: VimeoUpload<UploadDescriptor>
+class OldVimeoUploader: VimeoUploader<OldUploadDescriptor>
```

Comment:
Something looks backwards here, 

Although I haven't been in this code in a while / there were loose ends related to iOS7 / Cameo / VimeoNetworking. 

I can do a pass once merged to make sure that demo projects are working / compiling.